### DISCUSSION TITLE
Welcome to msn-js Discussions!

### DISCUSSION THREAD

**Discussion by pablobion** [pablobion/msn-js#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

---

## 👋  Bem-vindo

Estamos usando as discussões como um lugar para nos conectarmos com outros membros de nossa comunidade. Esperamos que você:

- Faça perguntas sobre as quais você está se perguntando.
- Compartilhar ideias.
- Envolva-se com outros membros da comunidade.
- Dê as boas-vindas aos outros e tenha a mente aberta. Lembre-se de que esta é uma comunidade que
- construímos juntos💪.

Para começar, comente abaixo apresentando-se e conte-nos o que você faz com esta comunidade.

**Comment by correaa1** [comment]
Que bacana
